src/anomalib/data/utils/label.py

```diff
@@ -0,0 +1,8 @@
+from enum import Enum
+
+
+class LabelType(int, Enum):
```

Comment:
How about `LabelName`? 

```suggestion
class LabelName(int, Enum):
```